### ISSUE TITLE
How emojis should appear

### ISSUE THREAD

**Issue by Kuladeep1996** [signalapp/Signal-Android#12339]
- [ x ] I have searched open and closed issues for duplicates
- [ x ] I am submitting a bug report for existing functionality that does not work as intended
- [ x ] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ x ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Emoji only messages used to be bigger on android before but after v5.41.14 update it looks smaller . In desktop it looks bigger and fine only in my android phone it looks smaller
### Steps to reproduce
- Send emoji only as message to android phone

**Actual result:** Emoji looks smaller
**Expected result:** Emoji should look bigger like before

### Screenshots
![pic](https://user-images.githubusercontent.com/74226018/177926165-8242c2e7-e4f1-40ff-a183-ee772395a82f.jpg)

### Device info

**Device:** Realme 7
**Android version:** 11
**Signal version:** v5.42.7

**Comment by cody-signal** [comment]
Do you have auto-download images disabled? We have to fetch the larger ones and try to respect your bandwidth settings.

**Comment by Kuladeep1996** [comment]
Yes I have disabled auto-download images, and after enabling, now emojis look bigger like before.
Haven't seen anywhere this feature is tied with auto-download images. If possible text below auto-download images (Something like emojis look bigger) works or a separate option to make emojis bigger also works. It helps others
